frontend/src/tabs/zoom.pug

```diff
@@ -39,8 +39,8 @@
     button(v-if="expandedCommitMessagesCount === 0" v-on:click="toggleAllCommitMessagesBody(true)") show all commit messages body
     button(v-else v-on:click="toggleAllCommitMessagesBody(false)") hide all commit messages body
 
-  .zoom__day(v-for="day in filteredUser.commits")
-    template(v-if="day.insertions > 0")
```

Comment:
https://github.com/reposense/RepoSense/blob/master/frontend/src/static/js/v_zoom.js#L27

This line should also be modified right?